gobblin-yarn/src/main/java/gobblin/yarn/YarnHelixUtils.java

```diff
@@ -199,4 +206,48 @@ public static void addFileAsLocalResource(FileSystem fs, Path destFilePath, Loca
 
     return environmentVariableMap;
   }
-}
+
+  public static Optional<Integer> takePort(int desiredPort) {
```

Comment:
Yes I think you are right: https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers#Registered_ports

Its still probably better to make sure the specified port is above 1024